python/GangaLHCb/Lib/Applications/GaudiExecUtils.py

```diff
@@ -99,7 +99,6 @@ def prepare_cmake_app(myApp, myVer, myPath='$HOME/cmtuser', myGetpack=None):
     full_path = expandfilename(myPath, True)
     if not path.exists(full_path):
         makedirs(full_path)
-        chdir(full_path)
```

Comment:
My reason for thinking it isn't needed is that the current working directory isn't changed when `$HOME/cmtuser` is already present on the filesystem.